praw/models/reddit/subreddit.py

```diff
@@ -553,7 +553,7 @@ def sticky(self, number=1):
         return self._submission_class(self._reddit, url=urljoin(
             self._reddit.config.reddit_url, path))
 
-    def submit(self, title, selftext=None, url=None, flair_id=None,
+    def submit(self, title, selftext=None, url=None, flair_id=None, nsfw=False,
```

Comment:
What do you think about adding this parameter at the end of the parameter list?

While the order _shouldn't_ matter as people should use these as keyword arguments, it might be nice to not introduce this as a breaking change in the event someone is using positional arguments.

Reply:
Oh, that's a good point. Yeah, I'll do that

Reply:
Fixed in https://github.com/praw-dev/praw/pull/1030/commits/9481cc3fa7f62fb3fe8a3f8e05ade6316ddbec26